commands/FBPrintCommands.py

```diff
@@ -299,3 +300,15 @@ def run(self, arguments, options):
       object = fb.evaluateObjectExpression(objectToMessage)
       printCommand = 'po [{} valueForKeyPath:@"{}"]'.format(object, keypath)
       lldb.debugger.HandleCommand(printCommand)
+
+
+class FBPrintApplicationDocumentsPath(fb.FBCommand):
+  def name(self):
+    return 'ppath'
```

Comment:
Thought: have `ppath` capable of printing other paths other than docs, and have it take an argument to specify which specific path. Otherwise, perhaps `ppath` can just print the app root, and then it's easy for users to append Documents or any other subpath.